LensFlare shaders errors

When opening the `highway.wbt` sample simulation (or any other world with LensFlare) I get these errors:
```
stefi@libeccio:~/webots$ ./webots projects/vehicles/worlds/highway.wbt 
An error occured during shader compilation/linking!
Vertex shader path: /home/stefi/webots/resources/wren/shaders/lens_flare.vert
Fragment shader path: /home/stefi/webots/resources/wren/shaders/lens_flare.frag
webots-bin: PostProcessingEffect.cpp:108: void wren::PostProcessingEffect::Pass::apply(): Assertion `locationTexture >= 0' failed.
./webots: line 88: 27958 Aborted                 (core dumped) "$webots_home/bin/webots-bin" "$@"
```

It works fine on the master branch.

I am getting a different warning on Windows. In the Webots console I get:
```
ERROR: Shader compilation failed! Filename: lens_flare.vert
0(6) : error C7614: GLSL ES doesn't allow use of reserved word noperspective
```
Which gives me some hints... I will investigate...

--- a/resources/wren/shaders/lens_flare.frag
+++ b/resources/wren/shaders/lens_flare.frag
@@ -4,7 +4,7 @@ precision highp float;
 
 const int lastResultTextureIndex = 0;
 
-noperspective in vec2 texUv;
+in vec2 texUv;
 
 layout(location = 0) out vec4 fragColor;
 

--- a/resources/wren/shaders/lens_flare.vert
+++ b/resources/wren/shaders/lens_flare.vert
@@ -3,7 +3,7 @@
 layout(location = 0) in vec3 vCoord;
 layout(location = 2) in vec2 vTexCoord;
 
-noperspective out vec2 texUv;
+out vec2 texUv;
 
 void main() {
   texUv = vTexCoord;

